Tensile/BenchmarkProblems.py

```diff
@@ -497,10 +502,16 @@ def writeBenchmarkFiles(stepBaseDir, solutions, problemSizes, stepName, filesToC
         kernelNames.add(kName)
     solutionKernelsBetaOnly = solution.getKernelsBetaOnly()
     for kernel in solutionKernelsBetaOnly:
-      kName = Solution.getNameFull(kernel)
+      kName = KernelWriter.getKernelNameBetaOnly(kernel)
       if kName not in kernelNamesBetaOnly:
         kernelsBetaOnly.append(kernel)
         kernelNamesBetaOnly.add(kName)
+    solutionKernelsGlobalAccum = solution.getKernelsGlobalAccum()
+    for kernel in solutionKernelsGlobalAccum:
```

Comment:
Could we generalize the helper kernels here instead of creating another special case?  That is, could we group the BetaOnly and GlobalAccum kernels as 'helper' kernels so we don't have to have special logic in so many places?

Reply:
Hi @sdquiring,

May you describe more detail about generalize kernel??

Purpose of BetaOnly and GlobalAccum is different, so they have different source body.
We need different function or class to generate their source code.

We can put kernel name or instance of Beta/GlobalAccum together, but for kernel generator, we have to use different method to handle it.

Thanks,
Tony.

Reply:
@sdquiring,

What do you think about putting generalization to another event???